sql/2019/04_Media/04_03.sql

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
BigQuery doesn't support this comment syntax for interpreting the style as standardSQL. Must be `#standardSQL`.